consensus/src/processes/transaction_validator/transaction_validator_populated.rs

```diff
@@ -48,10 +49,11 @@ impl TransactionValidator {
             // Storage mass hardfork was activated
             self.check_mass_commitment(tx)?;
 
-            // TODO: Decide what to do here
-            // if pov_daa_score < self.storage_mass_activation + 10 && self.storage_mass_activation > 0 {
-            //     warn!("--------- Storage mass hardfork was activated successfully!!! --------- (DAA score: {})", pov_daa_score);
-            // }
+            if self.storage_mass_activation.is_within_range_from_activation(pov_daa_score, 10)
+                && self.storage_mass_activation != ForkActivation::always()
```

Comment:
I think this condition can go within the is_within_range_from_activation call (+ a short doc explaining the behavior). Because 
1. activation can be defined as switching on something that was off, so "within range from activation" can logically be interpreted as false in the always case
2. I think all future calls to this method will require the same condition.